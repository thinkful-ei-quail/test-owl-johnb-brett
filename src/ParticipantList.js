import React from 'react'
import Participant from './Participant'

function ParticipantList() {
    return (
        <div>
        <Participant name='brett' inSession onStage={false}/>
        <Participant name='john' inSession={false} onStage={false} />
        <Participant name='bob' inSession={false} onStage={false} />
        </div>
    )
}

export default ParticipantList
