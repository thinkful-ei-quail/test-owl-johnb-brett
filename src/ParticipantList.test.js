import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import ParticipantList from './ParticipantList'

it('should render without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ParticipantList participants={[]}/>, div)
    ReactDOM.unmountComponentAtNode(div) 
})

it('renders UI as expected', () => {
    const tree = renderer
        .create(<ParticipantList participants={[]} />)
        .toJSON()
    expect(tree).toMatchSnapshot()
})