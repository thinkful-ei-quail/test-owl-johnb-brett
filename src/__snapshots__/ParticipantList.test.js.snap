// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UI as expected 1`] = `
<div>
  <div>
    <img
      alt="avatar"
    />
    <h3>
      brett
    </h3>
    <span
      style={
        Object {
          "color": "green",
          "fontSize": 50,
        }
      }
    >
      •
    </span>
    <span>
      observing
    </span>
  </div>
  <div>
    <img
      alt="avatar"
    />
    <h3>
      john
    </h3>
    <span
      style={
        Object {
          "color": "lightgrey",
          "fontSize": 50,
        }
      }
    >
      •
    </span>
    <span>
      observing
    </span>
  </div>
  <div>
    <img
      alt="avatar"
    />
    <h3>
      bob
    </h3>
    <span
      style={
        Object {
          "color": "lightgrey",
          "fontSize": 50,
        }
      }
    >
      •
    </span>
    <span>
      observing
    </span>
  </div>
</div>
`;
